### DISCUSSION TITLE
能不能在 idea 的 markdown 预览的时候进行翻译？

### DISCUSSION THREAD

**Discussion by mcxinyu** [YiiGuxing/TranslationPlugin#5876] [Ideas]
能不能在 idea 的 markdown 预览的时候进行翻译？

例如能不能在下面两个预览方式中的任何一个实现翻译功能呢？

![image](https://github.com/YiiGuxing/TranslationPlugin/assets/9566116/3b72f3e5-4e44-4010-a87b-5e96487eee6a)

**Comment by YiiGuxing** [comment]
暂时还不能
